firewood/src/merkle/node/leaf.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE.md for licensing terms.
+
+use std::fmt::{Debug, Error as FmtError, Formatter};
+
+use bincode::Options;
+
+use super::{Data, Encoded};
+use crate::merkle::{from_nibbles, PartialPath};
+
+#[derive(PartialEq, Eq, Clone)]
+pub struct LeafNode {
```

Comment:
nit: LeafNode shouldn't escape the crate, so the inner ones can just be pub, but this one should be `pub(crate)`.

Reply:
agreed, added to https://github.com/ava-labs/firewood/issues/362